Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -664,8 +727,11 @@ namespace Terrain
 
                 const float height = heights.at(queryCoord);
                 const float clampedHeight = AZ::GetClamp(height * rcpWorldZ, 0.0f, 1.0f);
-                const uint16_t uint16Height = aznumeric_cast<uint16_t>(clampedHeight * maxUint16 + 0.5f); // always positive, so just add 0.5 to round.
-                meshHeights.at(coord) = uint16Height;
+				
+				// For continuous LOD, it needs to be possible to create a height that's exactly in between any other height, so scale to 15 bits
+				// instead of 16, then multiply by 2, ensuring there's always an in-between value available.
+                const uint16_t uint16Height = aznumeric_cast<uint16_t>(clampedHeight * maxUint15 + 0.5f); // always positive, so just add 0.5 to round.
+                meshHeights.at(coord) = uint16Height * 2;
```

Comment:
Instead of scaling by 15 bits and rounding, I think you could also just scale by 16 bits and do "& 0xFFFE" to strip off the last bit?

Reply:
You could do it that way... I think to round correctly you'd add one? Not sure that way's any better though, just an alternate way to do the same thing.